### ISSUE TITLE
pip install stalls ; fails to awsume  a role

### ISSUE THREAD

**Issue by CJAclines** [trek10inc/awsume#19]
On windows 7 when installing using `pip install awsume` install would hang, and never finish.  After this it would recognize the command `awsume <profile>` but would not reflect the change if you do `aws sts get-caller-identity`. After looking found that .profile was missing in C:\Users\<username>. After creating file, and adding `alias awsume=". awsume"` it now appears to be working correctly

**Comment by charlesguse** [comment]
I would like to mention that this issue looks to be specific to mingw64.

**Comment by mbarneyjr** [comment]
Can you give me more information on your environment, such as what version of mingw64 you're using, or any other information about your environment?

I'm having a bit of trouble recreating the issue.

**Comment by CJAclines** [comment]
mingw64 was installed via git 2.14.2.windows.1 . Not sure what other environment information that would be effecting this.

**Comment by mbarneyjr** [comment]
I managed to recreate the problem, and identified a couple of other issues with some other features being incompatible with Git Bash / MINGW64. I've updated awsume to resolve those issues. Awsume version 2.1.4 should resolve these issues.

As for the `pip install awsume` hanging, the install did hang for me too, but continued when I gave the terminal an update by pressing `Enter`. I wasn't able to find out why this occurs, but a simple terminal update will resolve the issue.

Thanks for the bug report!
